### PR TITLE
Performance optimization for sshStorage

### PR DESCRIPTION
**Pull request by Riscue**
Concurrent Writes enabled

Suggested in https://github.com/pkg/sftp/issues/472#issuecomment-930022883

Uploads are 50x faster then before.

4 GiB file takes 5 hours before concurrent writes (~200KB/s). Now it takes like 10 mins (~10MB/s) on my ISP

### CODE REVIEW

**Review by m90 — COMMENTED** [review]
This looks very promising, thank you. I left one tiny remark, other than that let's release this.

File: internal/storage/ssh/ssh.go [hunk 1]
```diff
@@ -81,7 +81,11 @@ func NewStorageBackend(opts Config, logFunc storage.Log) (storage.Backend, error
 		return nil, err
 	}
 
-	sftpClient, err := sftp.NewClient(sshClient)
+	sftpClient, err := sftp.NewClient(sshClient,
+																		sftp.UseConcurrentReads(true),
```
[the remark this answers is not in the document]
Comment (Riscue) [reply internal/storage/ssh/ssh.go]: done

**Comment by Riscue** [comment]
> This looks very promising, thank you. I left one tiny remark, other than that let's release this.

Done. You're welcome.

**Comment by m90** [comment]
This is now released in [v2.34.0](https://github.com/offen/docker-volume-backup/releases/tag/v2.34.0) :tada:

**Comment by Riscue** [comment]
> This is now released in [v2.34.0](https://github.com/offen/docker-volume-backup/releases/tag/v2.34.0) 🎉

Thank you

**Pull request merged**